content/en/real_user_monitoring/guide/understanding-the-rum-event-hierarchy.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Understanding the RUM Event Hierarchy
+kind: guide
+further_reading:
+- link: '/real_user_monitoring/explorer/'
+  tag: 'Documentation'
+  text: 'Learn about the RUM Explorer'
+- link: '/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn how to visualize your RUM data'
+---
+
+## Overview
+
+This guide walks through the different [types of data][1] that RUM collects and describes the hierarchy of each event type. See the diagram below for a high level overview of the hierarchy.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-session-hierarchy-overview.png" alt="Diagram of the RUM event hierarchy." style="width:50%;">}}
+
+## Sessions
+All RUM data refers to user or synthetics sessions, which are at the top of the event hierarchy. A session is a unique user journey and encompasses everything (e.g., pages viewed, views, clicks, scrolls, errors) the user triggered. A session can last up to 4 hours of continuous activity, or it can expire after [15 minutes of inactivity][2]. Since a session encompasses the entire journey, all [attributes][3] tied to that user are also tied to that session. For example, you may want to query on a default attribute, like `application.id`, then add something more custom, like [user attributes][4].
+
+#### Sample search: List all sessions from user "Jane Doe"
+
+To list all sessions from a specific user, select **Sessions** from the event type dropdown, then make a search query for the user and application.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-sample-search-list-all-sessions.png" alt="Sample search listing all sessions from user "Jane Doe"." style="width:80%;">}}
```

Comment:
It would be great if we could make this example more modern and update the name being used here to something other than Jane Doe, which is used for unidentified dead bodies. Not a blocker but, ideally we'd use something else. (Nonbinary names are also nice!))